app/internal/security/dependancies.py

```diff
@@ -21,5 +25,32 @@ async def is_manager(
     """
     A dependency function protecting routes for only logged in manager
     """
-    await check_jwt_token(db, jwt, manager=True)
-    return True
+    jwt_payload = await get_jwt_token(db, jwt)
+    if jwt_payload.get("is_manager"):
+        return True
+    raise HTTPException(
+                status_code=HTTP_401_UNAUTHORIZED,
+                headers=request.url.path,
+                detail="You don't have a permition to enter this page")
+
+
+async def current_user(
```

Comment:
"current_user" -> "get_current_user"

Reply:
That is a dependency function, and not a function that anyone supposed to call directly. After many discussions, this is the name that was decided for this dependency.